docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -128,17 +128,17 @@ The following changes are supported by the adapter.
 ```diff
 import { createMuiTheme } from '@material-ui/core/styles';
 
--const theme = createMuitheme(),
-+const theme = createMuitheme({
+-const theme = createMuiTheme(),
++const theme = createMuiTheme({
 +  palette: { text: { hint: 'rgba(0, 0, 0, 0.38)' } },
 +});
 ```
 
 ```diff
 import { createMuiTheme } from '@material-ui/core/styles';
 
--const theme = createMuitheme({palette: { type: 'dark' }}),
-+const theme = createMuitheme({
+-const theme = createMuiTheme({palette: { type: 'dark' }}),
++const theme = createMuiTheme({
 +  palette: { type: 'dark', text: { hint: 'rgba(0, 0, 0, 0.38)' } },
```

Comment:
Wasn't sure whether to change this from `type` to `mode`.

Reply:
This should be changed, I think.

Reply:
Agree, we could base the diff based the most accurate state of v5. So use mode: dark in both cases. @hubgit do you want to send a pull request? :)

Reply:
I can add it to #23414 if you like?

Reply:
Ok, I'll send a PR. It seems that this diff is the wrong way round, too, if I'm understanding the changes correctly?

Reply:
The diff is correct

Reply:
Created #23416